cloud_storage.tf

```
@@ -0,0 +1,15 @@
+resource "google_storage_bucket" "storage_bucket" {
+  project = var.project_id
+  name = var.cloud_storage_name
+  location = var.location
```

```suggestion
  location = var.region
```

Change it also in `variables.tf` and `firestore.tf`.